app/javascript/components/supplemental_application/sections/PreferencesTable.js

```diff
@@ -121,7 +125,14 @@ const PreferencesTable = ({
     dispatch
   ] = useAppContext()
 
-  const rows = buildRows(application, applicationMembers, fileBaseUrl)
+  const sortedAndFilteredPreferences = sortAndFilter(application.preferences)
+
+  const rows = buildRows(
+    sortedAndFilteredPreferences,
+    application.proof_files,
+    applicationMembers,
+    fileBaseUrl
+  )
```

Comment:
question: For all of the changes made above this line, was the purpose just to be able to pass the `originals={sortedAndFilteredPreferences}` prop to the `sortedAndFilteredPreferences` component?

Reply:
Yes. By the time we call `ExpandableTable` the rows passed in only contain a subset of the original data, not including the preference type. I noticed that `ExpandableTable` accepts an `originals` prop which can later be called.